docs/Converters/UKBtoBIDS.md

```diff
@@ -0,0 +1,87 @@
+<!-- markdownlint-disable MD046 -->
+# `ukb-to-bids` – Conversion of the UK Biobank (UKB) to BIDS
+
+!!! quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
+    UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine and treatment and has enabled several scientific discoveries that improve human health
+
+## Dependencies
+
+If you installed the core of Clinica, this converter needs no further dependencies.
+
+## Downloading UK Biobank
+
+The UK Biobank to BIDS converter requires the user to have downloaded the UK Biobank imaging and clinical data.
+
+Once you have access to it, the imaging data can be downloaded and extracted using this tutorial: https://biobank.ctsu.ox.ac.uk/~bbdatan/Accessing_UKB_data_v2.3.pdf, and the clinical data can be extracted using the information from this repository https://github.com/kenhanscombe/ukbtools.
+
+
+Be careful about two things: 
+    - the converter only takes in the nifti provided by UK Biobank, not the dicoms.
+    - the converter will not work if you do not have among the clinical data the year of birth, the age at recruitment and at sessions and the sex of the subject.
```

Comment:
This seems difficult for a user to check. Maybe just add this message as an error message in the code if we can not find them ?

Reply:
I think it is basic data to have (not required by bids but always required by any study). And not very hard to check either ! Do you still want me to add a message in the code ?

Reply:
Let's place ourselves in the perspective of our users. First, they rarely reach for documentation *first*, only in case there is an issue (oftentimes) or when they are curious (sometimes). Second, imagine I run the converter on a datatset freshly downloaded from UKB and get a stack trace. Where do I get from here? Errors are usually for *exceptional* circumstances, not specific invariants that were not met that we deliberatly know of.

> the converter will not work if you do not have among the clinical data the year of birth, the age at recruitment and at sessions and the sex of the subject.

In my opinion, this is an invariant (or constraint) of our converter. Given any downloaded UKB datasets, we can determine whether these attributes are available as a validation step, prior to the conversion. So I'd expect a validation error with an appropriate message be thrown early in the execution, rather than a stack trace complaining of a missing `age` attribute in some DataFrame deep within the converter's logic.

Does it make sense to you?